routes/authentication.js

```diff
@@ -0,0 +1,100 @@
+const express = require('express');
+const router = express.Router();
+const passport   = require('passport');
+var multer  = require('multer');
+const { ensureLoggedIn, ensureLoggedOut } = require('connect-ensure-login');
+const User = require('../models/user');
+var upload = multer({ dest: './public/uploads/' });
+const bodyParser         = require('body-parser');
+
+
+// LOGIN
+
+router.get('/login', ensureLoggedOut(), (req, res, next) => {
+    res.render('authentication/login', { message: req.flash('error')});
+});
+
+router.post('/login', ensureLoggedOut(), (req, res, next) => {
+  let username = req.body.username;
+  console.log("DEBUG: " + username);
+  User.findOne({"username": username}, (err, user) => {
+    console.log("DEBUUUUUG " + user)
+    res.render('authentication/profile', {user})
```

Comment:
A better way here would have been to call res.redirect, sending the user to a new url which is his profile, and then handle a GET request to that url seperately. Notice that at the moment, the user is still in "/login" when he sees his profile, and this is not very scalable.